apps/bootgatthrm/boot.js

```diff
@@ -23,11 +39,11 @@
       NRF.updateServices({
         0x180D: {
           0x2A37: {
-            value: [
-              0x06, //
-              hrm.bpm
-            ],
+            value: [0x06, hrm.bpm],
             notify: true
+          },
+          0x2A38: {
```

Comment:
@gfwilliams Do I need to specify **all** values within a updateServices call, or only changed values?

Reply:
Only changed values :)